frontend/components/login/LoginProviders.tsx

```diff
@@ -7,19 +7,55 @@ import Link from 'next/link'
 import List from '@mui/material/List'
 import ListItem from '@mui/material/ListItem'
 import ListItemText from '@mui/material/ListItemText'
+import Chip from '@mui/material/Chip'
 
 import {Provider} from 'pages/api/fe/auth'
+import useRsdSettings from '~/config/useRsdSettings'
 
 type LoginProvidersProps=Readonly<{
   providers: Provider[]
   login_info_url?:string
 }>
 
+function SecondaryProviderText({provider}:Readonly<{provider:Provider}>){
+  const {host} = useRsdSettings()
+
+  if (provider?.html && host.email && provider?.accessType==='INVITE_ONLY'){
+    return (
+      <>
+        <span
+          dangerouslySetInnerHTML={{__html: provider.html}}
+        />
+        <span> Contact us at {host.email} for the invitation.</span>
+      </>
+    )
+  }
+  if (provider?.html){
+    return (
+      <span
+        dangerouslySetInnerHTML={{__html: provider.html}}
+      />
+    )
+  }
+  if (host.email && provider?.accessType==='INVITE_ONLY'){
```

Comment:
```suggestion
  if (host.email && provider?.accessType==='INVITE ONLY'){
```

I would prefer to replace `_` with space as `INVITE_ONLY` appears somewhat like debug output.